### ISSUE TITLE
Material set conflicts with Persian/Hebrew unicode chars

### ISSUE THREAD

**Issue by faridcher** [ryanoasis/nerd-fonts#478]
Hi,

I am trying to patch a font with Persian glyphs [Vazir](https://github.com/rastikerdar/vazir-code-font) with all of Nerd fonts suite of glyphs. I am on Linux with Gnome terminal. Vazir font is already merged with Inconsolita. I get the following message during merging:

```sh
./font-patcher -c --careful Vazir-Code.ttf

Adding 2119 Glyphs from Material Set 
73%  Found existing Glyph at FB01. Skipping...
73%  Found existing Glyph at FB02. Skipping...
77%  Found existing Glyph at FB56. Skipping...
77%  Found existing Glyph at FB57. Skipping...
77%  Found existing Glyph at FB58. Skipping...
77%  Found existing Glyph at FB59. Skipping...
78%  Found existing Glyph at FB7A. Skipping...
78%  Found existing Glyph at FB7B. Skipping...
78%  Found existing Glyph at FB7C. Skipping...
78%  Found existing Glyph at FB7D. Skipping...
79%  Found existing Glyph at FB8A. Skipping...
79%  Found existing Glyph at FB8B. Skipping...
79%  Found existing Glyph at FB8E. Skipping...
79%  Found existing Glyph at FB8F. Skipping...
79%  Found existing Glyph at FB90. Skipping...
79%  Found existing Glyph at FB91. Skipping...
79%  Found existing Glyph at FB92. Skipping...
79%  Found existing Glyph at FB93. Skipping...
80%  Found existing Glyph at FB94. Skipping...
80%  Found existing Glyph at FB95. Skipping...
80%  Found existing Glyph at FB9E. Skipping...
80%  Found existing Glyph at FB9F. Skipping...
80%  Found existing Glyph at FBA5. Skipping...
81%  Found existing Glyph at FBAC. Skipping...
81%  Found existing Glyph at FBAD. Skipping...
83%  Found existing Glyph at FBE8. Skipping...
84%  Found existing Glyph at FBE9. Skipping...
84%  Found existing Glyph at FBFC. Skipping...
84%  Found existing Glyph at FBFD. Skipping...
85%  Found existing Glyph at FBFE. Skipping...
85%  Found existing Glyph at FBFF. Skipping...
```

These unicode characters, excluding fi (FB01) and fl (FB02), are the following which are already provided by Vazir (or any other Persian/Arabic/Hebrew font):

`ﭖ ﭗ ﭘ ﭙ ﭺ ﭻ ﭼ ﭽ ﮊ ﮋ ﮎ ﮏ ﮐ ﮑ ﮒ ﮓ ﮔ ﮕ ﮞ ﮟ ﮥ ﮬ ﮭ ﯨ ﯩ ﯼ ﯽ ﯾ ﯿ`

And their equivalent in "Nerd Font Complete.ttf" are:

![image](https://user-images.githubusercontent.com/9982519/82384778-d8445800-9a45-11ea-82ac-77f5d1e4dbb9.png)

They conflict with Material set once it is merged into "Symbols-1000-em Nerd Font Complete.ttf" and their unicode positions are changed; they don't conflict with Material set itself but the "Nerd Font Complete.ttf".

I want to free from the `--careful` option and patch Vazir with complete sets of Nerd fonts and not to miss any glyphs from Material set.

**Comment by be5invis** [comment]
So NERD's added glyphs overflows PUA and overwrites [Arabic Presentation Forms-A](https://en.wikipedia.org/wiki/Arabic_Presentation_Forms-A) block?
In Unicode plane 15 and 16 (U+Fxxxx and U+10xxxx) are S-PUA planes which can definently hold all these icons.

**Comment by faridcher** [comment]
Arabic presentation Forms-A block also deploys [SPUA-A](https://en.wikipedia.org/wiki/Plane_(Unicode)) which I think NERD should carefully avoid the conflicts.

**Comment by be5invis** [comment]
@faridcher  Arabic Presentation Forms-A is defined to `U+FB50`..`U+FDFF`. SPUA-A is purely used for PUA. Some fonts may use them, but it will be safe to remove these mappings and replace with Nerd glyphs.

**Comment by aaronbell** [comment]
This has been brought up on several other threads: #365, #346. The maintainers know they need to move Material Design to a new location to avoid conflicts with the Hebrew and Arabic Presentation forms. However, it is some time off.

**Comment by terminaldweller** [comment]
are there any updates on this?

**Comment by Finii** [comment]
:thought_balloon:  Delugia :heart: 

Whatever, Nerd Fonts is 'super slow' because it is not only 1 font but 52 :-(

And anyhow, we did progress with the problem, the latest release `v2.3.3` comes with the Material Design Icons at both, the wrong/conflicting codepoints and a brand new correct codepoint range
* #773 

So now you can self-patch with `--careful` and preserve the Persion/Hebrew glyphs.
In fact, that is how it's done in latest Delugia ;)

With `v3.0.0` we will drop the old codepoints and `careful` will be no longer needed.

This is to give people a smoother way to change their codepoints.

So, maybe end of February we will have `v3.0.0` and finally freed the codepoints we stole accidentally years ago.

I close this now already, as self patching is already possible with no MDI loss.
